### PR TITLE
Add csi-driver-smoke job

### PR DESCRIPTION
**Pull request by kshlm**

### CODE REVIEW

**Comment by obnoxxx** [comment]
Why bother using the more heavyweight, expensive, and compllicated centos-ci for lightweight smoke-tests if we could use travis? See how e.g. heketi splits the tests between travis and centos...

File: csi-driver-smoke.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        - Madhu-1
+        white-list:
+        - humblec
+        - Madhu-1
+        cron: H/5 * * * *
+        status-context: centos-ci
+
+    builders:
+    - shell: !include-raw: scripts/common/get-node.sh
+    - shell: scripts/common/bootstrap.sh $WORKSPACE/scripts/csi-driver-smoke/run-test.sh "ghprbPullId=${ghprbPullId}"
```
Comment (obnoxxx) [inline csi-driver-smoke.yml]: I don't like the fact that the script to run those test comes from out of this repo. This is not clean. It should be under the control of the people working on the csi-driver itself. See for instance the heketi integration. The ci runner scripts reside in their own repo https://github.com/heketi/ci-tests which is referenced in the ci-job: https://github.com/gluster/centosci/blob/master/heketi-functional.yml#L24 . The scripts referenced to run and bootstrap the jobs are in that heketi repo. So the devs of the project to be tested do not always need to beg for some time of the admins of this repo when they need to change how tests are run or which...

File: scripts/csi-driver-smoke/run-test.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```
Comment (obnoxxx) [inline scripts/csi-driver-smoke/run-test.sh]: IMHO this should be in a different repo, either directly in the csi-driver repo or a related one.

Comment (kshlm) [reply scripts/csi-driver-smoke/run-test.sh]: The actual test runner script is in the csi-driver repo. This script just checks-out the csi-driver repo and the relevant PR and then runs the test runner from the checkout.

**Pull request merged**